public/kcl-samples-manifest-fallback.json

```diff
@@ -1 +1,212 @@
-404: Not Found
```

Comment:
Also making sure this gets reset properly, seems like we merged it and shouldn't have.